front-end/yarn.lock

```diff
@@ -6095,7 +6090,7 @@ debug@3.1.0:
   dependencies:
     ms "2.0.0"
 
-debug@^3.0.0, debug@^3.1.0, debug@^3.1.1, debug@^3.2.5, debug@^3.2.6:
+debug@^3.0.0, debug@^3.1.0, debug@^3.1.1, debug@^3.2.5:
```

Comment:
why so many changes in yarn.lock when package.json is unchanged. Can you run 

```
npx yarn-deduplicate ./yarn.lock
```

And check diff

Reply:
Can't really tell either tbh. removed the `yarn.lock` ran yarn again, it's updated packages mostly and duplicated one `fsevents@2.1.2` that `yarn-deduplicate` doesn't catch. I checked `yarn analyze` and there's nothing big, still at 7.5Mb chunks.

Reply:
it doesn't catch it because a higher version is available 2.1.3. `npx yarn-deduplicate -s fewer ./yarn.lock` does catch it, but that's not the strategy I'd go for.